js/privatebin.js

```diff
@@ -3000,7 +3026,9 @@ jQuery.PrivateBin = (function($, RawDeflate) {
 
             if (typeof fileName !== 'undefined') {
                 attachmentLink.attr('download', fileName);
-                template.append(fileName);
+
+                const fileSize = me.getAttachmentSize(attachmentData);
```

Comment:
We already have the `decodedData` available here on line 3022 above, why not reuse this or even better, just pass the `decodedData.length` to the `Helper.formatBytes` function and eliminate the getAttachmentSize one?
```suggestion
                const fileSize = Helper.formatBytes(decodedData.length);
```

Reply:
My logic is that we might need to get the size of a file in multiple places in the project, like for this issue #783, and it's better to have some helper function that we can pass a base64 encoded file to and get its size, like we have a function for getting the mime type
https://github.com/PrivateBin/PrivateBin/blob/409af9b9915b70d35b2ff206c8ab266cd3d63524/js/privatebin.js#L3196-L3203